### PR TITLE
Rename `reviews` to `ratings` internally

### LINKED ISSUE #6761: Rename reviews to ratings
Both internally and in the API, we should rename `reviews` to `ratings` to avoid clashing with `reviewers` (see #6751). For the API we should keep things backwards-compatible for now.

- [x] Move the app (and templates, fixtures etc)
- [x] Change the URL names
- [x] Change the model names
- [x] Change functions/methods/classes names (API only)

### PR DESCRIPTION
**Pull request by diox**
Fix #6761

### CODE REVIEW

File: src/olympia/activity/tests/test_serializers.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ def test_sanitized_activity_detail_not_exposed_to_developer(self):
         # Comments should be the santized text rather than the actual content.
         assert result['comments'] == amo.LOG.REQUEST_SUPER_REVIEW.sanitize
         assert result['comments'].startswith(
-            'The addon has been flagged for Admin Review.')
+            'The addon has been flagged for Admin Rating.')
```
Comment (eviljeff) [inline src/olympia/activity/tests/test_serializers.py]: This is a reviewer review :)  - Admin Review being the alternate name for Super Review

File: src/olympia/constants/activity.py [hunk 2]
```diff
@@ -202,7 +202,7 @@ class REQUEST_SUPER_REVIEW(_LOG):
     short = _(u'Super review requested')
     keep = True
     review_queue = True
-    sanitize = _(u'The addon has been flagged for Admin Review.  It\'s still '
+    sanitize = _(u'The addon has been flagged for Admin Rating.  It\'s still '
```
Comment (eviljeff) [inline src/olympia/constants/activity.py]: Same here - its a reviewer review

File: src/olympia/activity/tests/test_serializers.py (hunk 1 above)
Comment (diox) [reply src/olympia/activity/tests/test_serializers.py]: woops, overzealous search&replace, I kinda assume that all instances of `Review` with the capital R were references to the model...

**Comment by eviljeff** [comment]
`rating.rating` reads weirdly.  If I was designed from scratch I'd have `ratings.score` or something, but not sure it's worth more changes.

We should plan how to do the db table name change (along with editors) without everything breaking at once though.

**Comment by diox** [comment]
I can rename `rating.rating` in the model at least. I've avoided any database changes for now as they are trickier...

**Comment by EnTeQuAk** [comment]
> The question I'd like answered is: how far do we want to go? I've renamed the model (keeping the database table intact) in the 3rd commit, but is that a good idea? Should we rename all variables (like review = or reviews = everywhere as well?) Is ratings a good name to begin with?

Imho, "ratings" is a good name. In theory I'd go as far as to rename everything including the database but that is a hell lot of work. But, if we are going down this road we should have cleaner code as our dream-target since all this work is, is essentially improving code quality. And imho code quality only get's improved if we are thorough with this to avoid random future confusion.

Generally, I'm 👍 for this work, it'll make things a lot clearer!

We should rename things in the database as well. Simple renames should be relatively easy I hope, potentially even non blocking with recent mysql versions. Renaming indices and foreign keys probably is a hell lot more work which will get easier once we are on Django 1.11 😢

**Comment by diox** [comment]
I apologize for the huge patch, but this is now ready for a proper review. I've avoided making any changes to public URLs / parameter for now, so the API should behave like before and as a result, I've kept the `rating` field as-is for now.

**Comment by diox** [comment]
ping @eviljeff and @EnTeQuAk for a new look

File: src/olympia/ratings/models.py [hunk 3]
```diff
@@ -143,34 +143,34 @@ def delete(self, user_responsible=None):
...
+        log.info(u'Rating deleted: %s deleted id:%s by %s ("%s": "%s")',
                  user_responsible.name, self.pk, self.user.name,
                  unicode(self.title), unicode(self.body))
         self.update(deleted=True)
         # Force refreshing of denormalized data (it wouldn't happen otherwise
         # because we're not dealing with a creation).
         self.update_denormalized_fields()
 
-        if (review_was_moderated):
+        if (rating_was_moderated):
```
Comment (EnTeQuAk) [inline src/olympia/ratings/models.py]: This probably doesn't need parenthesis

**Pull request merged**